emva1288/tests/utils/logger.py

```diff
@@ -0,0 +1,25 @@
+import os
+import logging
+
+from root import ROOT_DIR
+
+
+class Logger:
+    def __init__(self, log_file):
+        self.logger = logging.getLogger(log_file)
+        self.logger.setLevel(logging.INFO)
+
+        formatter = logging.Formatter('%(asctime)s;%(levelname)s;%(message)s', '%Y-%m-%d %H:%M:%S')
+
+        self.log_file = log_file
+        self.log_file_path = os.path.join(ROOT_DIR, 'emva1288/tests/logs/' + self.log_file)
```

Comment:
As the logs folder is not present by default in tests folder, running the tests gives error by default

Reply:
Indeed you are correct, I addressed this in comment above ;)

Reply:
Other option could be to just use tests folder for log.

Reply:
True , I have updated the Logger Class with this validation and it handles the issue.

if not os.path.exists(logs):
    os.makedirs(logs)

 I suppose its personal preference, but I like to keep logs in a separate folder. Especially, as testing grows and logs for 
 different tests are written to seperate log files.